client/me/help/help-contact/index.jsx

```diff
@@ -256,15 +257,59 @@ class HelpContact extends React.Component {
 	};
 
 	submitSupportForumsTopic = ( contactForm ) => {
-		const { subject, message } = contactForm;
-		const { currentUserLocale } = this.props;
+		const {
+			helpSiteIsJetpack,
+			helpSiteIsNotWpCom,
+			site,
+			subject,
+			message,
+			userDeclaresNoSite,
+			userDeclaredUrl,
+			userRequestsHidingUrl,
+		} = contactForm;
+		const { currentUserLocale, translate } = this.props;
 
 		this.setState( { isSubmitting: true } );
 		this.recordCompactSubmit( 'forums' );
 
+		let blogHelpMessage = translate( "I don't have a site linked to this WordPress.com account" );
```

Comment:
Should we really translate these messages to the user's selected language? Or should all forum posts be English?

Reply:
I'll check this again to be certain, but I think that depending on the user's selected language, the question will be posted in a Forum for their language, so this should be translated. For example, the French forum contains numerous threads sent from this form: https://wordpress.com/fr/forums/topic-tag/wpcomhelp/

Reply:
If a user's account language matches a language for which we have a support forum, their requests are automatically sent to the appropriate language forum, afaik.

Public list of our non-English forums can be found at https://wordpress.com/forums/topic/forums-in-other-languages-not-english/

Reply:
What if I submit a question in a language that doesn't have a dedicated forum? In Czech, for example, the Help form would be localized. But will there be anyone to undestand my question? Does it happen in practice?